src/snowflake/snowpark/modin/pandas/general.py

```diff
@@ -2714,6 +2716,104 @@ def date_range(
     return pd.DatetimeIndex(query_compiler=qc)
 
 
+def bdate_range(
+    start=None,
+    end=None,
+    periods: int | None = None,
+    freq: Frequency | dt.timedelta = "B",
+    tz=None,
+    normalize: bool = True,
+    name: Hashable | None = None,
+    weekmask=None,
+    holidays=None,
+    inclusive: IntervalClosedType = "both",
+    **kwargs,
```

Comment:
Can we add type annotations here for all of the params? e.g. `tz: str | None = None`